docs/reference/openapi-rules.md

```diff
@@ -397,6 +397,44 @@ TheBadModel:
         - 8
 ```
 
+### duplicated-entry-in-enum
```

Comment:
@philsturgeon While reviewing the file I've sent, I saw that I didn't sent the right sample on this file --> Updated with good samples :)